akvo/rsr/management/commands/lock_unlock_periods.py

```diff
@@ -21,28 +21,40 @@ class Command(BaseCommand):
         make_option('-k', '--keyword',
```

Comment:
I can't get keywords to work. They can be more than one word and even if i wrap them in " only the first token is used: ```./manage.sh lock_unlock_periods unlock -k "EU Trust Fund - North of Africa"``` results in the following debug output:
```
(Pdb) options
{'keyword': 'EU', 'settings': None, 'pythonpath': None, 'verbosity': u'1', 'traceback': None, 'no_color': False, 'organisation': None}
```

Reply:
There are multiple problems here. 

1. The `manage.sh` script does some bash magic, and doesn't handle the args with spaces very well. Escape the spaces, or use `manage.py` from the vagrant box. 
2. They keyword you were trying to use has a unicode hyphen, it looks like `\u2013`.  Try using `'EU Trust Fund - Sahel and Lake Chad'`